src/pages/[...slug].tsx

```diff
@@ -46,49 +46,9 @@ export interface PageProps {
     page: PageWithMetadata
 }
 
-const HEADING_SELECTOR = 'h1, h2, h3, h4, h5, h6'
-
 export default function Page({ page }: PageProps): JSX.Element {
     const markdownBodyReference = useRef<HTMLElement>(null)
     const tocReference = useRef<HTMLElement>(null)
-    useEffect(() => {
```

Comment:
Imo even if we only show H2, we shouldn't remove the highlighting (bolding) of the current section. Imo it's a nice anchor of where you are in the document, like the "YOU ARE HERE" dot on a public map.